code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -198,6 +198,50 @@
 	playsound(user, 'sound/weapons/empty.ogg', 100, 1)
 	update_icon()
 
+/obj/item/gun/projectile/automatic/ak47
+	name = "\improper AK-47"
+	desc = "A relic from the USSP Cygni Rebellion era, its design remains unchanged hundreds of years later. Why would it need to be?"
+	icon_state = "ak47"
+	item_state = "ak47"
+	weapon_weight = WEAPON_MEDIUM //two handed handling recommended, it's a rifle
+	origin_tech = "combat=5;materials=3;syndicate=5"

```

Comment:
Syndicate 5 is an EXTREMELY high tech level. Illegal/Syndicate tech has a different scale than most other tech, I think it caps out at 3 for syndicate longarms.

Reply:
SMG and carbine are both Syndicate 6.

Reply:
I'm a dumbass, oops.

Reply:
Not actually that much of a dumbass because it's true it was too high, just for the wrong reasons :P I ended up lowering it to 2.